pkg/crypto/tinkcrypto/primitive/composite/ecdhes/subtle/ecdhes_aead_composite_encrypt.go

```diff
@@ -94,6 +98,7 @@ func (e *ECDHESAEADCompositeEncrypt) Encrypt(plaintext, aad []byte) ([]byte, err
 	tagOffset := len(ctAndTag) - tagSize
 
 	encData := &EncryptedData{
+		EncAlg:     A256GCM, // TODO support 25519 content encryption as well
```

Comment:
Actually, X25519 is for ECDH as an additional support beyond P256.

chacha is the additional content encryption algorithm

Reply:
sure, I'll update the comment to mention (X)Chacha20Poly1305 encryption alg..

Reply:
updated